contracts/FeeBurner.sol

```diff
@@ -90,11 +90,15 @@ contract FeeBurner is Withdrawable, FeeBurnerInterface, Utils2 {
         (kyberEthKncRate, ) = kyberNetwork.getExpectedRate(ETH_TOKEN_ADDRESS, ERC20(knc), (10 ** 18));
         (kyberKncEthRate, ) = kyberNetwork.getExpectedRate(ERC20(knc), ETH_TOKEN_ADDRESS, (10 ** 18));
 
+        // if returned rate is 0, can't trust this arb test. assume we have arb
+        require(kyberEthKncRate > 0);
```

Comment:
seems redundant to line 100
no?

Reply:
Ah, because it's covered in line 100. Agreed. Can remove then.